### PR TITLE
Feature/CVSB-840

### PR DESCRIPTION
**Pull request by teoAlexandru**
@lgvasile  @dorinNicolae please review my changes.

### CODE REVIEW

File: src/test/java/activities/PostActivities.java [hunk 1]
```diff
@@ -24,12 +26,57 @@
...
-        activitiesSteps.postActivities(ActivitiesData.buildActivitiesData().setActivityType("visit").build());
-        activitiesSteps.statusCodeShouldBe(201);
-        activitiesSteps.responseShouldContainId();
+        int i = 0;
+
+        while (true) {
+            i ++;
+            System.out.println("Iteration " + i);
+            activitiesSteps.postActivities(activitiesData.setActivityType("visit").build());
+//            activitiesSteps.postActivities(activitiesData.setActivityType(“visit”)
```
Comment (dorinNicolae) [inline src/test/java/activities/PostActivities.java]: Commented code

File: src/test/java/activities/PostActivities.java [hunk 2]
```diff
@@ -24,12 +26,57 @@
...
+            activitiesSteps.getActivities("visit", activitiesData.build().getTesterStaffId(), null, DataUtil.buildCurrentDateTime(-1), null);
+
+            statuscode = activitiesSteps.statusCodeShouldBe(200);
+            if (statuscode != 200) {
+                activitiesSteps.printStatusAndBody();
+                break;
+            }
+            activitiesSteps.validateData(activitiesData.build());
+
+//            try {
```
Comment (dorinNicolae) [inline src/test/java/activities/PostActivities.java]: Commented code

File: src/test/java/activities/PostActivities.java [hunk 3]
```diff
@@ -24,12 +26,57 @@
...
+            statuscode = activitiesSteps.statusCodeShouldBe(204);
+
+            if (statuscode != 204) {
+                activitiesSteps.printStatusAndBody();
+                break;
+            }
+        }
+
+
+//        activitiesSteps.postActivities(ActivitiesData.buildActivitiesData().setActivityType("visit").build());
```
Comment (dorinNicolae) [inline src/test/java/activities/PostActivities.java]: Commented code

File: src/test/java/activities/PostActivities.java [hunk 4]
```diff
@@ -24,12 +26,57 @@
     @Steps
     ActivitiesSteps activitiesSteps;
 
+    private ActivitiesGet.Builder activitiesData = ActivitiesData.buildActivitiesData();
+
     @Title("CVSB-163 / CVSB-2874 - AC8 API Consumer creates a new activity - activity type: visit")
     @Test
     public void postActivitiesActivityTypeVisit() {
```
Comment (lgvasile) [inline src/test/java/activities/PostActivities.java]: revert all changes here

File: src/main/java/steps/ActivitiesSteps.java [hunk 5]
```diff
@@ -103,4 +104,9 @@ public String checkAndGetResponseId() {
         return response.jsonPath().get("id");
     }
 
+    public void printStatusAndBody() {
```
Comment (lgvasile) [inline src/main/java/steps/ActivitiesSteps.java]: remove this

File: src/main/java/steps/ActivitiesSteps.java [hunk 6]
```diff
@@ -41,8 +41,9 @@ public void getActivities(String activityType, String testerStaffId, String test
     }
 
     @Step
-    public void statusCodeShouldBe(int statusCode) {
```
Comment (lgvasile) [inline src/main/java/steps/ActivitiesSteps.java]: revert this

File: src/test/java/testresults/PostTestResultsTestCodeMappingOnTestTypes.java [hunk 7]
```diff
@@ -0,0 +1,612 @@
...
+    TestResultsSteps testResultsSteps;
+
+    private TestResults.Builder vehicleSubmittedDataOne = TestResultsData.buildTestResultsSubmittedData();
+    private TestResults.Builder vehicleSubmittedDataTwo = TestResultsData.buildTestResultsSubmittedData();
+
+    private void validateSavedData(String... testCodes) {
+
+        testResultsSteps.getTestResults(vehicleSubmittedDataOne.build().getVin(), TestResultsStatus.SUBMITTED);
+        testResultsSteps.statusCodeShouldBe(200);
+//        testResultsSteps.validateData((TestResultsGet) vehicleSubmittedDataOne.build());
```
Comment (lgvasile) [inline src/test/java/testresults/PostTestResultsTestCodeMappingOnTestTypes.java]: remove comented

File: src/main/java/steps/ActivitiesSteps.java [hunk 8]
```diff
@@ -103,4 +104,9 @@ public String checkAndGetResponseId() {
         return response.jsonPath().get("id");
     }
 
+    public void printStatusAndBody() {
+        System.out.println("status code is " + response.getStatusCode() + " and body is:");
```
Comment (dorinNicolae) [inline src/main/java/steps/ActivitiesSteps.java]: remove system out

File: src/test/java/activities/PostActivities.java (hunk 1 above)
Comment (teoAlexandru) [reply src/test/java/activities/PostActivities.java]: reverted

File: src/test/java/activities/PostActivities.java (hunk 3 above)
Comment (teoAlexandru) [reply src/test/java/activities/PostActivities.java]: reverted

File: src/main/java/steps/ActivitiesSteps.java [hunk 9]
```diff
@@ -103,4 +104,9 @@ public String checkAndGetResponseId() {
...
+        response.prettyPrint();
```
Comment (dorinNicolae) [inline src/main/java/steps/ActivitiesSteps.java]: remove pretty print

File: src/test/java/activities/PostActivities.java (hunk 4 above)
Comment (teoAlexandru) [reply src/test/java/activities/PostActivities.java]: reverted

File: src/test/java/activities/PostActivities.java (hunk 2 above)
Comment (teoAlexandru) [reply src/test/java/activities/PostActivities.java]: reverted

File: src/main/java/steps/ActivitiesSteps.java (hunk 9 above)
Comment (teoAlexandru) [reply src/main/java/steps/ActivitiesSteps.java]: removed

File: src/main/java/steps/ActivitiesSteps.java (hunk 8 above)
Comment (teoAlexandru) [reply src/main/java/steps/ActivitiesSteps.java]: removed

File: src/main/java/steps/ActivitiesSteps.java (hunk 5 above)
Comment (teoAlexandru) [reply src/main/java/steps/ActivitiesSteps.java]: resolved

File: src/test/java/testresults/PostTestResultsTestCodeMappingOnTestTypes.java [hunk 10]
```diff
@@ -0,0 +1,612 @@
...
+                .setNoOfAxles(2).build()
+                .getTestTypes().get(0).setTestTypeId("1");
+
+        testResultsSteps.postTestResults(vehicleSubmittedDataOne.build());
+        testResultsSteps.statusCodeShouldBe(201);
+        testResultsSteps.validateData("Test records created");
+        validateSavedData("aal");
+    }
+
+    @Test
```
Comment (lgvasile) [inline src/test/java/testresults/PostTestResultsTestCodeMappingOnTestTypes.java]: add Title

Comment (teoAlexandru) [reply src/test/java/testresults/PostTestResultsTestCodeMappingOnTestTypes.java]: done

**Pull request merged**